setuptools/command/bdist_wheel.py

```diff
@@ -291,10 +291,15 @@ def _validate_py_limited_api(self) -> None:
             raise ValueError(f"py-limited-api must match '{PY_LIMITED_API_PATTERN}'")
 
         if sysconfig.get_config_var("Py_GIL_DISABLED"):
+            # sys.abiflags is only defined on POSIX
+            if hasattr(sys, "abiflags"):
+                abinote = f" ({sys.abiflags=!r}). "
+            else:
+                abinote = ". "
             raise ValueError(
                 f"`py_limited_api={self.py_limited_api!r}` not supported. "
                 "`Py_LIMITED_API` is currently incompatible with "
-                f"`Py_GIL_DISABLED` ({sys.abiflags=!r}). "
+                f"`Py_GIL_DISABLED`{abinote}"
```

Comment:
Suggestion from https://github.com/pypa/setuptools/pull/4809#issuecomment-2602590023

```suggestion
                "`Py_GIL_DISABLED`."
```